src/Altinn.App.Core/Features/Telemetry.AppOptionsService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Diagnostics;
+using Altinn.App.Core.Models;
+
+namespace Altinn.App.Core.Features;
+
+partial class Telemetry
+{
+    internal Activity? StartGetOptionsAsyncActivity() =>
+        ActivitySource.StartActivity("AppOptionsService.GetOptionsAsync");
+
+    internal Activity? StartGetOptionsAsyncActivity(InstanceIdentifier instanceIdentifier)
+    {
+        var activity = ActivitySource.StartActivity("AppOptionsService.GetOptionsAsync");
```

Comment:
Here the span name matches the method name, which sort of introduces coupling (if we change the method name, we should change span name as well). Should span names be more "logical names"/descriptive of the feature or should they match technical method names? Async suffix or not?